src/components/JobListings/components/JobListing/JobListing.tsx

```diff
@@ -29,7 +29,7 @@ export function JobListing({ job, addFilter, removeFilter, filters }: Props) {
         alt="logo"
         className="-mt-[25px] md:mt-0 w-12 md:w-[80px] md:h-[80px]"
       />
-      <div>
+      <div className="min-w-[225px]">
```

Comment:
Notced that there are still some elements that stack with min-w-[225px]...I experimented with some settings and found that flex-[0_1_100%] worked better...
![Screen Shot 2023-01-31 at 7 06 55 PM](https://user-images.githubusercontent.com/9553453/215921746-cd206d1c-ced9-4bf1-bfe5-b3a48d3d0171.png)

Reply:
If you update the div and replace min-w with  flex-[0_1_120%]  I'll approve the updates and push up my additional changes in the next pr

Reply:
I have made the necessary changes, you can check.